keyboards/rmkeebs/rm_fullsize/info.json

```
@@ -0,0 +1,156 @@
+{
+    "manufacturer": "RuckerMachine",
+    "keyboard_name": "rm_fullsize",
+    "maintainer": "RuckerMachine",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "encoder_map": true
```

Encoder map should only ever be enabled at the keymap level.
```suggestion
        "encoder": true
```